src/Types/TypeProps.tsx

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
I would change the name of this folder to `typings` since that's common practice

Reply:
If you change the file extension to `.d.ts` (type definition file), TypeScript will automatically make all these types global to your project, and you want have to import/export any of the types.